### ISSUE TITLE
NDK headers

### ISSUE THREAD

**Issue by louwers** [bazelbuild/rules_android_ndk#55]
How do I get access to NDK headers `android/log.h`, `android_native_app_glue.h`, etc.

This does not seem to be documented.

**Comment by louwers** [comment]
I cannot really try it out, because I abandoned trying to build our Android library with Bazel.

**Comment by ted-xie** [comment]
Sorry to hear that! In case you'd be interested in trying again in the future, we've made a lot of progress in major feature work for both rules_android and rules_android_ndk in the last year.

I also tried porting one of the example apps under `ndk-samples` to use Bazel + rules_android_ndk. Unfortunately that app was pure native, and I don't think rules_android supports that, but I was able to get pretty far, at least to the point where the C++ native deps were compiled: https://github.com/ted-xie/ndk-samples/commit/01f438dc73bac0141e39eae390d29a182e162cb4
